packages/core/src/Stores/client-store.js

```diff
@@ -2505,8 +2505,12 @@ export default class ClientStore extends BaseStore {
     get is_high_risk() {
         if (isEmptyObject(this.account_status)) return false;
         const { gaming_company, financial_company } = this.landing_companies;
+
+        // This is a conditional check for countries like Australia/Norway which fulfil one of these following conditions.
+        const restricted_countries = financial_company?.shortcode === 'svg' || gaming_company?.shortcode === 'svg';
```

Comment:
the first and second line should be removed
```suggestion
        const restricted_countries = financial_company?.shortcode === 'svg' || (gaming_company?.shortcode === 'svg' && financial_company?.shortcode !== 'maltainvest')`;
```